docs/install/sandboxes.rst

```diff
@@ -315,3 +315,12 @@ the ``FROM`` line in any dockerfile.
 
 This will be particularly useful if you are interested in modifying envoy, and testing
 your changes.
+
+
+Using alpine envoy image
+------------------------
```

Comment:
can you actually add this info here: https://github.com/lyft/envoy/blob/11faea44b5b1c8ca8d868856190e65f2aaf64a9e/ci/README.md#developer-use-of-ci-docker-image